src/libraries/System.Diagnostics.EventLog/ref/System.Diagnostics.EventLog.csproj

```diff
@@ -10,7 +10,7 @@
     <ProjectReference Include="$(LibrariesProjectRoot)System.Security.Permissions\ref\System.Security.Permissions.csproj" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
-    <ProjectReference Include="$(LibrariesProjectRoot)System.Diagnostics.TraceSource\src\System.Diagnostics.TraceSource.csproj" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Diagnostics.TraceSource\ref\System.Diagnostics.TraceSource.csproj" />
```

Comment:
Is there any way we can guard against folks making this mistake?  Did you scan for other instances of it?

Reply:
Filed https://github.com/dotnet/runtime/issues/56428

Reply:
Scanned for other hits. There aren't any.

Reply:
Hmm, I accidentally only searched for "\src\" but not for "/src/". Found one more hit.

Reply:
Fond 12 hits with:

```
git grep -E 'ProjectReference.*[\\/]src[\\/](Microsoft.Bcl.AsyncInterfaces|System.Diagnostics.TraceSource).csproj'
```

not sure if they are all related though.. 😅

Reply:
Check out this sample as a way that you could validate this: https://github.com/ericstj/runtime/commit/66dca09639305ec25cfcd694ce82af0ef85088b4

Reply:
I searched for `/src/` and `\src\` in all `src/libraries/*/ref/*.*proj` so I'm pretty sure that I found all 😄